torchvision/transforms/functional_tensor.py

```diff
@@ -183,6 +196,7 @@ def five_crop(img, size):
 
 
 def ten_crop(img, size, vertical_flip=False):
+    # type: (Tensor, BroadcastingList2[int], bool) -> List[Tensor]
```

Comment:
Do we return a list of tensors or a tuple of tensors here? Maybe we might want to cast explicitly to `List` in the return value?